devboard/maixbit/examples/pwm/main.go

```diff
@@ -21,7 +21,7 @@ var tickCount uint64
 //go:interrupthandler
 func TIMER0A_Handler() {
```

Comment:
Consider move the interrupt handler at the end of this file (all examples with interrupts are written this way).

Consider also replace `tickCount++` with a comment like *there you can do something useful on every tick*.

Alternatively you can print tickCount value in `for range tick.C {` loop. Don't leave it unused. It can confuse people there is something unobvious in this code that they don't understand.